src/Catty/Defines/NetworkDefinesSwift.swift

```diff
@@ -38,6 +38,8 @@ let kFeaturedImageBaseUrl = kBaseUrl.replacingOccurrences(of: "/pocketcode/", wi
 let kChartProjectsMaxResults = 10
 let kRecentProjectsMaxResults = 20
 let kSearchStoreMaxResults = 50
+let kAspectRatioHeight: CGFloat = 25
```

Comment:
Thanks for this fix. Do you think it would be possible to remove the hard-coded aspect ratio and directly calculate it from the image dimensions? That would have the big advantage that we do not need to redeploy when the image dimensions should change on the server-side.

Reply:
Yes, you are right, but I think the implementation would be too time consuming as the image is cached via the Kingfisher library in a separate file.